pkg/operator/workloads/api.go

```diff
@@ -253,10 +333,19 @@ func apiWorkloadSpecs(ctx *context.Context) ([]*WorkloadSpec, error) {
 			}
 		}
 
+		var spec metav1.Object
+
+		if api.ModelType == userconfig.TensorFlowModelType {
```

Comment:
nit: switch statement might be a little clearer. In the `default` case, you can return an internal error (which for now just means that I haven't been adding error types, and instead just inline the string)